### PR TITLE
session 2

### PR DESCRIPTION
**Pull request by tharun634**

### CODE REVIEW

File: session_2/student_registry/app/models/student.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Student < ApplicationRecord
+  validates_presence_of :name, :roll_number, :branch, :address, :admission_year, :cgpa, message: 'cannot be null',
```
Comment (abhishekkumar2718) [inline session_2/student_registry/app/models/student.rb:2]: `validates_presence_of` is the old syntax.

Prefer using `validates :name, :roll_number, ..., presence: true`

File: session_2/student_registry/app/models/student.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+                                                                                        on: :create
```
Comment (abhishekkumar2718) [inline session_2/student_registry/app/models/student.rb:3]: The `on: :create` is incorrect - Can a student have a nil roll number after creation?

**Review by abhishekkumar2718 — CHANGES_REQUESTED** [review]

**Comment by abhishekkumar2718** [comment]
Noice, Noice, Noice! :partying_face:

File: session_2/student_registry/app/models/student.rb (hunk 2 above)
Comment (tharun634) [reply session_2/student_registry/app/models/student.rb:3]: so, it's better to leave it on default? that is, on save?

File: session_2/student_registry/app/models/student.rb (hunk 1 above)
Comment (tharun634) [reply session_2/student_registry/app/models/student.rb:2]: alright, noted! =))

File: session_2/student_registry/app/models/student.rb (hunk 2 above)
Comment (abhishekkumar2718) [reply session_2/student_registry/app/models/student.rb:3]: Yeah

**Pull request closed without merging**